examples/python-cicd-with-cloudbuilder/python-cloud-builder/cloudbuild.yaml

```diff
@@ -1,3 +1,7 @@
+# This cloudbuild.yaml is used to build a custom cloud builder, shown in step 3 of README.md
+# When you run gcloud builds submit --config=cloudbuild.yaml . in this directory it will use the yaml file
```

Comment:
Floating period in the middle.

Reply:
That's intentional and part of the command. I quoted it to be clearer.